### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by koterupanchajanyareddy**
@LaxmiNarayanaMurthyVemuri @rehana-ms

### CODE REVIEW

File: Project 1/project1/application.py [hunk 1]
```diff
@@ -38,20 +38,26 @@ def index():
...
+        first_name = data.get("fname")
+        last_name = data.get("lname")
+        email_id= data.get("email")
         password = data.get("password")
-        u = Users(first_name, last_name, email_id, password)
+        u = Users(first_name, last_name, email_id, password,)
         try:
             db.session.add(u)
         except:
             return render_template("signup.html", details = [username, email,"Failed to registered."])
```
Comment (rehana-ms) [inline Project 1/project1/application.py:49]: username,email variables are not defined.pls check once.

File: Project 1/project1/application.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        data = request.form
+        first_name = data.get("fname")
+        last_name = data.get("lname")
+        email_id= data.get("email")
+        password = data.get("password")
+        u = Users(first_name, last_name, email_id, password,)
+        try:
+            db.session.add(u)
+        except:
+            return render_template("signup.html", details = [username, email,"Failed to registered."])
```
Comment (rehana-ms) [inline Project 1/project1/application.py]: username,email variables are not defined.pls check once.

**Comment by koterupanchajanyareddy** [comment]
@LaxmiNarayanaMurthyVemuri @rehana-ms

**Comment by koterupanchajanyareddy** [comment]
@LaxmiNarayanaMurthyVemuri @rehana-ms  sign in and log out is also done. Please check it.

File: Project 1/project1/application.py [hunk 3]
```diff
@@ -38,20 +38,26 @@ def index():
...
     if request.method == "POST":
         data = request.form
-        first_name = data.get("first_name")
-        last_name = data.get("email_id")
+        first_name = data.get("fname")
+        last_name = data.get("lname")
+        email_id= data.get("email")
         password = data.get("password")
-        u = Users(first_name, last_name, email_id, password)
+        u = Users(first_name, last_name, email_id, password,)
```
Comment (rehana-ms) [inline Project 1/project1/application.py:50]: You have timestamp in Users model.Are you not storing it?

Comment (koterupanchajanyareddy) [reply Project 1/project1/application.py:50]: I am storing mam. When creating the User object itself in the constructor I have created and stored.

File: Project 1/project1/application.py (hunk 2 above)
Comment (koterupanchajanyareddy) [reply Project 1/project1/application.py]: I have already updated it mam.

**Review by rehana-ms — COMMENTED** [review]
Good to merge.

File: Project 1/project1/application.py (hunk 3 above)
Comment (rehana-ms) [reply Project 1/project1/application.py:50]: ok

File: Project 1/project1/application.py (hunk 2 above)
Comment (rehana-ms) [reply Project 1/project1/application.py]: ok.

**Pull request closed without merging**